samples/CommunityToolkit.Maui.Sample/Pages/Converters/NotEqualConverterPage.xaml.cs

```diff
@@ -1,11 +1,12 @@
-﻿namespace CommunityToolkit.Maui.Sample.Pages.Converters;
+﻿using CommunityToolkit.Maui.Sample.ViewModels.Converters;
 
-public partial class NotEqualConverterPage : BasePage
+namespace CommunityToolkit.Maui.Sample.Pages.Converters;
+
+public partial class NotEqualConverterPage : BasePage<NotEqualConverterViewModel>
 {
-	public NotEqualConverterPage()
+	public NotEqualConverterPage(NotEqualConverterViewModel notEqualConverterViewModel)
+		: base(notEqualConverterViewModel)
 	{
 		InitializeComponent();
-
-		ExampleText ??= new();
```

Comment:
Please check if it works on Windows

Reply:
In this case, I removed `x:Name:ExampleText` from the XAML, and the field `ExampleText` no longer exists. 

Last I checked, the `??= new()` workaround is still required on Windows fro Preview 13. Hopefully that'll be fixed in the RC! 🤞